distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -510,12 +542,24 @@ public void testExportFailureWithoutWSSecurityDirectory() throws Exception {
         String response = console.runCommand(EXPORT_COMMAND);
 
         assertThat(String.format("Should not have been able to export to %s.",
-                getDefaultExportDirectory()),
-                response,
+                getDefaultExportDirectory()), response,
                 containsString("An error was encountered while executing this command."));
     }
 
     /**
+     * Tests that when CRLs are enabled, they get exported
+     */
+    @Test
+    public void testExportCrlsEnabled() throws Exception {
+        resetInitialState();
```

Comment:
We can't use @Before on the resetInitialState method as there are a few existing tests that will fail if cleanup is performed.  These are the tests that we force junit to "run first".